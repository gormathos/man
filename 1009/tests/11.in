sol_id;user_id;task_id;score;posted_time
159697;1006;2010;76.67;1705833713
159698;1098;2007;34.09;1705834022
159700;1005;2007;45.45;1705835383
159703;1096;2004;100.00;1705835823
159707;1097;2005;90.48;1705838950
159712;1038;2004;21.95;1705843919
159718;1089;2002;50.00;1705844766
159725;1008;2009;65.00;1705850699
159733;1035;2005;100.00;1705853990
159742;1048;2004;90.24;1705857521
159752;1036;2008;6.12;1705861324
159763;1015;2008;100.00;1705864541
159775;1080;2005;100.00;1705870127
159788;1061;2010;66.67;1705874291
159802;1019;2002;56.52;1705878915
159817;1079;2008;4.08;1705881094
159833;1056;2002;65.22;1705886629
159850;1027;2004;100.00;1705888382
159868;1089;2009;75.00;1705893608
159887;1031;2001;100.00;1705899598
159907;1020;2003;45.83;1705905008
159928;1022;2008;79.59;1705906859
159950;1004;2010;100.00;1705909763
159973;1007;2007;93.18;1705909959
159997;1062;2004;90.24;1705915925
160022;1003;2008;53.06;1705917540
160048;1001;2001;100.00;1705919148
160075;1025;2008;100.00;1705923595
160103;1020;2009;85.00;1705924309
160132;1080;2004;75.61;1705924710
160162;1052;2001;94.00;1705926610
160193;1085;2009;15.00;1705932329
160225;1050;2005;85.71;1705937663
160258;1001;2003;37.50;1705938018
160292;1050;2001;92.00;1705939026
160327;1093;2002;100.00;1705942171
160363;1035;2007;59.09;1705943024
160400;1085;2009;80.00;1705945280
160438;1093;2008;71.43;1705947741
160477;1026;2005;100.00;1705948654
160517;1036;2010;100.00;1705948858
160558;1071;2001;38.00;1705952805
160600;1039;2008;67.35;1705954514
160643;1065;2003;100.00;1705959944
160687;1081;2007;68.18;1705962923
160732;1038;2008;28.57;1705966405
160778;1019;2010;26.67;1705966682
160825;1059;2005;100.00;1705972189
160873;1063;2007;0.00;1705973316
160922;1009;2004;36.59;1705974013
160972;1063;2004;4.88;1705978258
161023;1078;2008;81.63;1705979671
161075;1068;2008;100.00;1705981623
161128;1072;2010;100.00;1705983160
161182;1004;2002;100.00;1705986675
161237;1090;2003;54.17;1705987805
161293;1046;2007;36.36;1705989788
161350;1053;2009;95.00;1705992857
161408;1045;2003;100.00;1705996674
161467;1097;2008;100.00;1705997049
161527;1004;2001;100.00;1705998409
161588;1012;2004;58.54;1705999893
161650;1012;2004;31.71;1706002231
161713;1006;2005;90.48;1706006370
161777;1071;2002;86.96;1706011147
161842;1036;2001;100.00;1706015074
161908;1020;2010;56.67;1706020590
161975;1086;2005;100.00;1706023692
162043;1018;2006;28.57;1706028193
162112;1066;2003;100.00;1706032229
162182;1089;2010;50.00;1706035298
162253;1010;2007;18.18;1706039925
162325;1019;2007;100.00;1706041834
162398;1033;2008;0.00;1706046154
162472;1039;2004;21.95;1706048610
162547;1055;2006;53.57;1706052057
162623;1084;2003;70.83;1706054154
162700;1053;2004;68.29;1706054990
162778;1056;2003;33.33;1706056313
162857;1075;2008;95.92;1706059058
162937;1014;2010;100.00;1706059906
163018;1059;2009;40.00;1706064321
163100;1001;2005;23.81;1706069836
163183;1081;2010;0.00;1706073762
163267;1058;2005;4.76;1706076929
163352;1029;2005;76.19;1706077911
163438;1041;2002;100.00;1706078649
163525;1084;2005;57.14;1706081490
163613;1035;2003;100.00;1706082321
163702;1043;2002;30.43;1706087274
163792;1038;2004;14.63;1706088249
163883;1032;2001;100.00;1706090584
163975;1069;2010;70.00;1706094585
164068;1011;2007;100.00;1706100385
164162;1062;2006;39.29;1706101122
164257;1096;2002;71.74;1706105020
164353;1073;2009;100.00;1706107048
164450;1001;2003;8.33;1706108305
164548;1095;2002;36.96;1706112042
164647;1014;2002;73.91;1706112085
164747;1055;2005;33.33;1706117421
164848;1015;2005;85.71;1706117906
164950;1087;2006;75.00;1706118232
165053;1018;2005;47.62;1706120979
165157;1078;2003;25.00;1706126586
165262;1043;2006;50.00;1706127133
165368;1093;2007;100.00;1706129967
165475;1012;2003;100.00;1706133890
165583;1008;2005;100.00;1706139812
165692;1024;2002;100.00;1706141928
165802;1010;2003;100.00;1706144100
165913;1077;2004;41.46;1706147456
166025;1067;2002;54.35;1706150569
166138;1037;2005;0.00;1706156219
166252;1045;2001;22.00;1706160975
166367;1026;2003;100.00;1706164561
166483;1017;2008;95.92;1706168732
166600;1053;2001;100.00;1706172615
166718;1016;2004;21.95;1706176685
166837;1038;2007;9.09;1706181368
166957;1013;2009;100.00;1706184668
167078;1008;2005;19.05;1706185057
167200;1028;2009;40.00;1706189240
167323;1035;2005;100.00;1706190506
167447;1088;2008;59.18;1706194242
167572;1081;2003;100.00;1706197960
167698;1097;2004;97.56;1706198357
167825;1092;2009;100.00;1706201540
167953;1081;2006;100.00;1706206523
168082;1006;2010;50.00;1706207234
168212;1016;2010;70.00;1706211832
168343;1025;2005;0.00;1706212653
168475;1009;2003;25.00;1706212712
168608;1095;2007;100.00;1706217722
168742;1099;2010;20.00;1706223318
168877;1038;2003;79.17;1706224914
169013;1059;2010;3.33;1706226835
169150;1031;2008;38.78;1706231653
169288;1080;2004;100.00;1706235997
169427;1044;2010;86.67;1706240550
169567;1089;2007;100.00;1706241133
169708;1022;2001;48.00;1706241763
169850;1078;2004;19.51;1706243787
169993;1092;2005;90.48;1706245069
170137;1022;2009;100.00;1706248627
170282;1029;2003;91.67;1706251429
170428;1090;2002;8.70;1706255141
170575;1007;2001;4.00;1706257640
170723;1077;2003;100.00;1706258646
170872;1089;2002;60.87;1706261171
171022;1020;2008;85.71;1706262020
171173;1039;2005;38.10;1706263997
171325;1017;2010;60.00;1706266192
171478;1024;2007;61.36;1706270918
171632;1064;2005;66.67;1706275187
171787;1032;2003;75.00;1706279999
171943;1067;2005;0.00;1706283322
172100;1039;2007;97.73;1706285614
172258;1045;2009;100.00;1706290648
172417;1008;2010;26.67;1706296299
172577;1030;2009;35.00;1706300165
172738;1054;2006;64.29;1706304814
172900;1065;2005;76.19;1706307870
173063;1008;2008;69.39;1706313231
173227;1048;2002;56.52;1706318660
173392;1015;2003;25.00;1706324111
173558;1003;2010;83.33;1706328217
173725;1072;2010;100.00;1706329016
173893;1093;2001;82.00;1706332810
174062;1007;2004;100.00;1706334873
174232;1028;2007;100.00;1706335403
174403;1003;2009;40.00;1706339985
174575;1055;2004;82.93;1706341729
174748;1011;2005;57.14;1706347283
174922;1027;2002;100.00;1706348527
175097;1033;2007;63.64;1706350386
175273;1089;2001;20.00;1706351142
175450;1089;2010;63.33;1706354191
175628;1009;2006;53.57;1706355465
175807;1076;2009;35.00;1706359540
175987;1023;2006;60.71;1706363180
176168;1004;2003;95.83;1706368364
176350;1077;2004;7.32;1706371035
176533;1055;2005;33.33;1706374342
176717;1037;2008;77.55;1706376400
176902;1028;2002;84.78;1706382370
177088;1088;2006;17.86;1706384477
177275;1025;2002;100.00;1706389357
177463;1052;2006;39.29;1706391003
177652;1007;2009;80.00;1706392533
177842;1066;2009;70.00;1706393292
178033;1043;2006;10.71;1706398826
178225;1056;2003;4.17;1706401918
178418;1081;2008;59.18;1706404300
178612;1100;2001;100.00;1706405546
178807;1042;2006;78.57;1706407880
179003;1020;2008;14.29;1706410140
179200;1042;2002;76.09;1706413554
179398;1049;2007;93.18;1706413954
179597;1014;2009;80.00;1706417536
179797;1073;2007;29.55;1706423291
179998;1070;2002;56.52;1706423662
180200;1077;2004;68.29;1706423668
180403;1070;2008;100.00;1706424936
180607;1087;2008;100.00;1706430753
180812;1060;2010;30.00;1706433922
181018;1093;2010;20.00;1706439893
181225;1025;2009;85.00;1706443457
181433;1078;2007;100.00;1706444247
181642;1099;2006;78.57;1706446026
181852;1016;2006;100.00;1706448733
182063;1003;2004;82.93;1706449945
182275;1062;2008;38.78;1706455785
182488;1091;2010;3.33;1706458524
182702;1051;2002;100.00;1706458918
182917;1082;2010;100.00;1706464814
183133;1003;2005;38.10;1706467906
183350;1078;2004;19.51;1706473120
183568;1034;2007;100.00;1706473459
183787;1038;2008;16.33;1706477581
184007;1082;2004;48.78;1706480640
184228;1068;2004;87.80;1706484510
184450;1045;2002;95.65;1706486409
184673;1037;2008;20.41;1706486522
184897;1059;2004;56.10;1706489575
185122;1008;2002;69.57;1706494151
185348;1041;2008;12.24;1706499047
185575;1047;2006;0.00;1706504366
185803;1077;2006;25.00;1706508738
186032;1016;2007;100.00;1706514100
186262;1044;2009;35.00;1706518243
186493;1043;2008;44.90;1706523456
186725;1017;2010;96.67;1706529090
186958;1092;2007;15.91;1706533274
187192;1036;2010;53.33;1706534990
187427;1076;2007;65.91;1706537107
187663;1083;2001;50.00;1706537250
187900;1047;2005;100.00;1706538586
188138;1032;2005;100.00;1706542015
188377;1081;2006;32.14;1706543532
188617;1024;2009;100.00;1706547884
188858;1082;2001;64.00;1706550064
189100;1001;2005;100.00;1706553996
189343;1050;2010;66.67;1706558936
189587;1018;2001;100.00;1706564209
189832;1080;2007;100.00;1706564669
190078;1095;2006;85.71;1706565021
190325;1053;2002;8.70;1706566482
190573;1075;2010;50.00;1706567116
190822;1036;2008;100.00;1706571468
191072;1082;2009;100.00;1706576139
191323;1053;2003;79.17;1706581503
191575;1090;2005;100.00;1706587233
191828;1055;2003;100.00;1706587553
192082;1025;2009;55.00;1706590083
192337;1002;2009;40.00;1706595193
192593;1072;2009;75.00;1706600942
192850;1029;2004;100.00;1706602255
193108;1090;2005;47.62;1706604114
193367;1051;2005;52.38;1706610091
193627;1088;2007;54.55;1706610370
193888;1093;2010;6.67;1706611236
194150;1012;2010;6.67;1706617189
194413;1079;2003;4.17;1706621984
194677;1067;2004;68.29;1706624289
194942;1042;2003;45.83;1706625998
195208;1036;2010;73.33;1706626614
195475;1033;2003;33.33;1706626777
195743;1022;2008;55.10;1706628727
196012;1086;2001;90.00;1706631869
196282;1080;2002;43.48;1706635237
196553;1100;2004;19.51;1706636986
196825;1083;2006;71.43;1706639539
197098;1011;2006;10.71;1706644486
197372;1051;2005;0.00;1706645226
197647;1072;2003;8.33;1706649833
197923;1027;2010;70.00;1706650551
198200;1005;2006;96.43;1706655999
198478;1039;2005;100.00;1706658710
198757;1079;2009;100.00;1706661873
199037;1095;2006;53.57;1706664735
199318;1077;2007;6.82;1706666720
199600;1074;2010;16.67;1706671143
199883;1032;2004;24.39;1706671465
200167;1074;2003;100.00;1706673596
200452;1010;2001;18.00;1706677006
200738;1006;2001;72.00;1706678394
201025;1055;2008;100.00;1706678710
201313;1033;2010;16.67;1706680436
201602;1094;2002;47.83;1706683518
201892;1017;2010;100.00;1706686299
202183;1043;2005;100.00;1706692218
202475;1024;2007;34.09;1706695221
202768;1073;2007;100.00;1706699259
203062;1065;2005;66.67;1706704733
203357;1087;2009;55.00;1706710305
203653;1010;2008;28.57;1706715828
203950;1071;2007;59.09;1706718307
204248;1061;2010;83.33;1706723370
204547;1046;2002;60.87;1706728195
204847;1001;2004;87.80;1706728708
205148;1045;2004;100.00;1706732876
205450;1086;2003;0.00;1706735803
205753;1003;2009;65.00;1706737189
206057;1059;2005;9.52;1706737498
206362;1016;2004;39.02;1706738907
206668;1043;2008;20.41;1706742730
206975;1089;2009;40.00;1706745155
207283;1042;2004;80.49;1706748163
207592;1021;2007;40.91;1706753678
207902;1077;2002;36.96;1706758777
208213;1036;2002;47.83;1706759234
208525;1093;2001;100.00;1706763497
208838;1054;2003;29.17;1706765793
