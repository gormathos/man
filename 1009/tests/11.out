82
21.01.2024 11:17:03
21.01.2024 16:19:50
21.01.2024 19:15:41
21.01.2024 20:48:47
22.01.2024 01:53:02
22.01.2024 04:59:58
22.01.2024 07:49:23
22.01.2024 10:25:48
22.01.2024 11:39:55
22.01.2024 16:49:31
22.01.2024 18:37:34
22.01.2024 18:40:58
22.01.2024 21:45:44
23.01.2024 01:09:49
23.01.2024 03:47:03
23.01.2024 04:12:40
23.01.2024 05:11:15
23.01.2024 07:57:54
23.01.2024 08:04:09
23.01.2024 08:26:49
23.01.2024 13:04:34
23.01.2024 15:28:12
23.01.2024 17:50:29
23.01.2024 20:30:34
24.01.2024 01:31:46
24.01.2024 06:44:09
24.01.2024 07:45:21
24.01.2024 10:03:04
24.01.2024 12:46:25
24.01.2024 14:37:28
24.01.2024 20:59:27
24.01.2024 22:04:50
24.01.2024 23:43:32
25.01.2024 00:18:48
25.01.2024 00:55:00
25.01.2024 06:36:01
25.01.2024 08:50:15
25.01.2024 12:11:08
25.01.2024 13:48:26
25.01.2024 15:52:40
25.01.2024 16:52:20
25.01.2024 18:15:23
25.01.2024 21:22:02
26.01.2024 02:26:37
26.01.2024 03:52:13
26.01.2024 05:57:07
26.01.2024 08:44:06
26.01.2024 17:37:28
27.01.2024 04:16:56
27.01.2024 05:54:33
27.01.2024 06:03:23
27.01.2024 09:42:07
27.01.2024 21:02:37
28.01.2024 01:32:26
28.01.2024 06:55:36
28.01.2024 08:32:33
28.01.2024 12:17:27
28.01.2024 13:32:13
28.01.2024 16:21:58
28.01.2024 18:00:14
28.01.2024 20:24:19
29.01.2024 07:41:40
29.01.2024 14:29:46
29.01.2024 15:26:55
29.01.2024 17:04:44
29.01.2024 18:46:36
29.01.2024 21:36:49
29.01.2024 21:44:29
29.01.2024 23:37:48
30.01.2024 00:55:39
30.01.2024 04:00:33
30.01.2024 04:05:53
30.01.2024 08:10:55
30.01.2024 23:51:50
31.01.2024 00:44:33
31.01.2024 03:59:56
31.01.2024 05:25:10
31.01.2024 07:31:39
31.01.2024 09:10:18
31.01.2024 11:07:39
31.01.2024 20:27:56
01.02.2024 04:58:17
